Networking/NetworkingTests/ProductImageInBackground/ProductImageStatusStorageTests.swift

```diff
@@ -0,0 +1,412 @@
+import XCTest
+import Combine
+@testable import Networking
+
+class ProductImageStatusStorageTests: XCTestCase {
+    private var cancellables = Set<AnyCancellable>()
+    private var userDefaults: UserDefaults!
+    private let userDefaultsKey: String = "productImageStatusStorageTests"
+    private var productImagesStatuses: ProductImageStatusStorage!
```

Comment:
I think renaming this into `storage` or `productImageStatusStorage` will increase the readability of the tests. 

```suggestion
    private var storage: ProductImageStatusStorage!
```

Reply:
Good suggestion, renamed here: acb630032168d3e8d14f7ea514b83fd7c18e7dc8